examples/notebooks/Watch_Warning_and_Advisory_Plotting.ipynb

```diff
@@ -0,0 +1,721 @@
+   "source": [
+    "## Function: get_color()\n",
+    "\n",
+    "Since we'll be needing to access the color using the vtec lookup table in several places, creating a easily recognizable function is useful."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_color(phensig):\n",
+    "    return vtec[phensig]['color']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Function get_title()\n",
+    "\n",
+    "Similar to the color function just defined, accessing the full name for the phensig will also be necessary, so this function will be helpful."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_title(phensig):\n",
+    "    return vtec[phensig]['hdln']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inital Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###  EDEX Connection\n",
+    "\n",
+    "First we establish a connection to Unidata's public EDEX server.  With that connection made, we can create a [new data request object](http://unidata.github.io/python-awips/api/IDataRequest.html) and set the data type to ***warning***, and set the Parameters to ***phensig*** and ***sig***.\n",
+    "\n",
+    "<br>\n",
+    "<div class=\"alert-info\">\n",
+    "<b>Note:</b> Remember, to see all available parameters use the <a href=\"http://unidata.github.io/python-awips/api/DataAccessLayer.html#awips.dataaccess.DataAccessLayer.getAvailableParameters\"><b>DataAccess.getAvailableParameters()</b></a> method as shown in the <a href=\"http://unidata.github.io/python-awips/examples/generated/Grid_Levels_and_Parameters.html#get-available-parameters\"><b>Grid Levels and Parameters Notebook.</b></a>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DataAccessLayer.changeEDEXHost(\"edex-cloud.unidata.ucar.edu\")\n",
+    "request = DataAccessLayer.newDataRequest()\n",
+    "request.setDatatype(\"warning\")\n",
+    "params = [\"phensig\", \"sig\"]\n",
+    "request.setParameters(*(params))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Significance (Sig) Constants\n",
+    "\n",
+    "The two parameters we're requesting for our warning objects are ***phensig*** and ***sig*** where phensig is styled \"XX.Y\" and sig is \"Y\".  The constants in this section correlate the ***sig*** to what type of message it is (what significance it is)."
```

Comment:
As we discussed earlier "phen" stands for "phenomenon", so I would mention that here.

Also would it make more sense to request the phen and sig parameters? And you could join them together later down in the code to get phensig? These two sentences just seem a little odd since both phensig and sig params have the significance. 

Here's the PDF we looked at: https://www.weather.gov/media/vtec/VTEC_explanation4-20.pdf

Reply:
Both the entire phensig and the sig are needed in the notebook.  It's much easier to just request and use phensig once, than parse it together with phen and sig.  Technically sig doesn't need to be requested since it can just be parsed from the phensig object....which I could change the code to do if we really think that's better?  
Initially the notebook only requested the phensig parameter, and I added sig when I noticed I needed...  It makes the data call slightly less efficient I suppose, but I thought it made the code easier to read and write?

Reply:
This is what I updated the language in this section to be, and added a link in our "additional docs" for it as well.
![image](https://user-images.githubusercontent.com/21114460/165636427-0fa4a73a-4443-47f7-9fc4-b95d5f3d8608.png)